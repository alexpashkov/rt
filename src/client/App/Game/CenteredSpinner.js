import React from 'react';
import { PacmanLoader } from 'react-spinners';

const CenteredSpinner = () => (
  <div
    css={`
      display: flex;
      align-items: center;
      justify-content: center;
      width: 100%;
    `}
  >
    <PacmanLoader color="red" />
  </div>
);

export default CenteredSpinner;
