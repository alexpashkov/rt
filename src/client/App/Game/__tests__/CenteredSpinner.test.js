import React from 'react';
import CenteredSpinner from '../CenteredSpinner';
import renderer from 'react-test-renderer';

describe('CenteredSpinner', () => {
  it('Renders without throwing an exception', () => {
    const component = renderer.create(<CenteredSpinner />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
