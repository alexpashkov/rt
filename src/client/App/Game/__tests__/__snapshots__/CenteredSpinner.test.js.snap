// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CenteredSpinner Renders without throwing an exception 1`] = `
<div
  className="css-1lads1q"
>
  <div
    className="css-ph4qhg"
  >
    <div
      className="css-16uiyca"
    />
    <div
      className="css-nh4zb1"
    />
    <div
      className="css-lqc6el"
    />
    <div
      className="css-15k2385"
    />
    <div
      className="css-1p90qyq"
    />
    <div
      className="css-1rg2f2b"
    />
  </div>
</div>
`;
